ssh/tailssh/tailssh.go

```diff
@@ -332,6 +334,8 @@ func (srv *server) endSession(s ssh.Session) {
 	delete(srv.activeSessions, s.Context().(ssh.Context).SessionID())
 }
 
+var recordSSH = envknob.Bool("TS_LOG_SSH")
```

Comment:
Make this part of the action?

Reply:
I should name this TS_TEMP_DEBUG_LOG_SSH or something; the real recording mechanism will stream off-device. Otherwise it's not secure having the machine being sshed into also storing the recordings.

This whole PR is just a baby step towards the mechanism itself but isn't the final product.